en/sensor/rpm.md

```diff
@@ -0,0 +1,40 @@
+# ThunderFly TFRPM01 tachometer sensor
+
+The [TFRPM01](https://github.com/ThunderFly-aerospace/TFRPM01) is a small, and low system demanding revolution-counter.
+It must be connected to a I²C bus.
+
+![TFRPM01A](../..//assets/hardware/sensors/tfrpm/tfrpm01_electronics.jpg)
+
```

Comment:
I think at this point it is also worth adding a note that it is generic counter that must be connected to a specific sensor instance - then maybe link down to examples